boards/arm/mr_canhubk3/mr_canhubk3.dts

```diff
@@ -409,10 +409,18 @@
 	phy-connection-type = "rmii";
 	local-mac-address = [02 04 9f aa bb cc];
 	status = "okay";
+};
 
-	fixed-link {
-		speed = <100>;
-		full-duplex;
+&mdio0 {
```

Comment:
please add a mention in the boards doc

- add an entry for `mdio` API under GMAC in https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mr_canhubk3/doc/index.rst?plain=1#L59
- update the Ethernet related information to mention there's a driver that overrides the PHY pin strapping settings to use it in "slave" mode https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mr_canhubk3/doc/index.rst?plain=1#L254-L257

Reply:
Added.